css/index.css

```diff
@@ -104,14 +122,84 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.box:nth-of-type(1) {background: teal}
```

Comment:
Love the way you did your CSS for styling the boxes here! Very clean and DRY! Great Job!